micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -2,66 +1,65 @@
+  const additionalParams = { from: from, screen: screen, action: action }
   Object.keys(additionalParams).forEach(key => {
     if (additionalParams[key] === undefined || additionalParams[key] === null) {
       delete additionalParams[key];
     }
   });
-  // const stateId = Digit.ULBService.getStateId();
-  tenantId = Digit.ULBService.getCurrentTenantId();
+
+
   const fetchActionItems = (data) => {
     let actionItems = [{
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'actionItems' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)